docs/source/algorithms/Q1DWeighted-v1.rst

```diff
@@ -69,6 +94,11 @@ The wedges output group will have two workspaces: one for the red region, one fo
   :align: center
   :width: 600
 
+In the case of drawn sectors, the symmetric computation will happen only if the pair of sectors has been provided,
```

Comment:
Does this mean that the flag `AsymmetricWedges` will also be ignored?
I think it makes more sense to still respect this.
Usually they will always have two back-to-back wedges.
Still, they might want to integrate either each one individually, or both together.